messages/teleporter/utils.go

```diff
@@ -47,3 +48,10 @@ func CalculateReceiveMessageGasLimit(numSigners int, executionRequiredGasLimit *
 
 	return res, nil
 }
+
+// Pack the SendCrossChainMessage event type. PackEvent is documented as not supporting struct types, so this should be used
+// with caution. Here, we only use it for testing purposes. In a real setting, the Teleporter contract should pack the event.
+func PackTeleporterMessage(destinationChainID common.Hash, message TeleporterMessage) ([]byte, error) {
```

Comment:
Similarly, does this need to be exported?

Reply:
this is used in `e2e_test.go`